modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java

```diff
@@ -238,6 +239,46 @@ public long getAccountGroupRelsCountByAccountGroupId(long accountGroupId) {
 		return accountGroupRelPersistence.countByAccountGroupId(accountGroupId);
 	}
 
+	private Predicate _getPredicate(
+		Long[] accountEntryIds, String className, long classPK,
+		String keywords) {
+
+		Predicate predicate = AccountGroupRelTable.INSTANCE.classNameId.eq(
+			_classNameLocalService.getClassNameId(className)
+		).and(
+			AccountGroupRelTable.INSTANCE.classPK.eq(classPK)
+		);
+
+		if (accountEntryIds.length > 0) {
```

Comment:
ArrayUtil.isEmpty (I think is something like this)

Reply:
no longer needed.